Source/WebKit2/PlatformWPE.cmake

```diff
@@ -477,7 +482,7 @@ set(WPE_INSTALLED_WEBKIT_HEADERS
     ${WEBKIT2_DIR}/UIProcess/API/C/WKProxy.h
     
     ${WEBKIT2_DIR}/UIProcess/API/C/wpe/WKView.h
-
+    ${WEBKIT2_DIR}/UIProcess/API/C/wpe/WKWebAutomation.h
```

Comment:
These entries should be grouped together by the directory they're in, and an empty line between the groups.